### ISSUE TITLE
[V3 Audio] Error on skip

### ISSUE THREAD

**Issue by laggron42** [Cog-Creators/Red-DiscordBot#1761]
# Other bugs

#### What were you trying to do?

Skip a song using the command

#### What were you expecting to happen?

Make the bot skip the song

#### What actually happened?

```Py
Exception in command 'skip'
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 62, in wrapped
    ret = yield from coro(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/redbot/cogs/audio/audio.py", line 1428, in skip
    return await self._skip_action(ctx)
  File "/usr/local/lib/python3.6/dist-packages/redbot/cogs/audio/audio.py", line 1492, in _skip_action
    pos, dur = player.position, player.current.length
AttributeError: 'NoneType' object has no attribute 'length'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/bot.py", line 886, in invoke
    yield from ctx.command.invoke(ctx)
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 514, in invoke
    yield from injected(*ctx.args, **ctx.kwargs)
  File "/usr/local/lib/python3.6/dist-packages/discord/ext/commands/core.py", line 71, in wrapped
    raise CommandInvokeError(e) from e
discord.ext.commands.errors.CommandInvokeError: Command raised an exception: AttributeError: 'NoneType' object has no attribute 'length'
```

#### How can we reproduce this issue?

Get a queue with less than one song, then skip

**Comment by tekulvw** [comment]
Was a song playing?

**Comment by aikaterna** [comment]
This is when a user with elevated audio privileges queues a song, the song/queue runs out, the person leaves the channel, and then uses [p]skip.

**Comment by tekulvw** [comment]
Fixed with #1769
